src/collective/mustread/interfaces.py

```diff
@@ -190,10 +192,91 @@ def who_unread(obj, force_deadline=True):
         If ``force_deadline`` is False, returns scheduled users who
         did not read the object at all.
 
+        @guido: i'd suggest to remove force_deadline as we have the get_stats
+        method to get a detailed report which users read and when
+
         :param obj: Object that should be read by users
         :type obj: Content object (must be IUUID resolvable)
         :param force_deadline: Whether to ignore reads after the deadline
         :type force_deadline: Bool
-        :returns: Userids of users scheduled for reading which did not read
-        :rtype: List
+        :returns: Dictionary with userids of users scheduled for reading which
+                  did not read and their deadline (``{userid: deadline}``)
```

Comment:
Here again you're working with the `deadline` concept, so I say keep `force_deadline` in the method signature for this method.